tools/docker/build.sh

```diff
@@ -1,37 +1,38 @@
-#!/bin/sh
+#!/bin/bash
 ###############################################################
 # SPDX-License-Identifier: BSD-2-Clause-Patent
 # SPDX-FileCopyrightText: 2019-2020 the prplMesh contributors (see AUTHORS.md)
 # This code is subject to the terms of the BSD+Patent license.
 # See LICENSE file for more details.
 ###############################################################
 
-scriptdir="$(cd "${0%/*}"; pwd)"
+scriptdir="$(cd "${0%/*}" || exit 1; pwd)"
 rootdir="${scriptdir%/*/*}"
 
-. ${rootdir}/tools/functions.sh
+# shellcheck source=../../tools/functions.sh
+. "${rootdir}/tools/functions.sh"
 
 main() {
     docker image inspect prplmesh-builder >/dev/null 2>&1 || {
         echo "Image prplmesh-build does not exist, creating..."
-        run ${scriptdir}/image-build.sh
+        run "${scriptdir}"/image-build.sh
     }
 
     # Default docker arguments
-    docker_args="\
-    --workdir=$rootdir --user=${SUDO_UID:-$(id -u)}:${SUDO_GID:-$(id -g)} \
-    -e USER=${SUDO_USER:-${USER}} -v ${rootdir}:${rootdir} \
-    --entrypoint=./tools/maptools.py \
-    "
+    docker_args=(
```

Comment:
(nitpick) Now we're using bash, we can also 
```suggestion
    local docker_args=(
```